terraform_discovery.py

```diff
@@ -35,48 +37,48 @@ def __init__(self, sc_params, slack_params, log):
 namespaces = []
 
 
-def update_sc_namespace(ns_id, data, services):
-  services.log.debug(f'Namespace data: {data}')
+def update_sc_namespace(ns_id, data):
+  globals.services.log.debug(f'Namespace data: {data}')
```

Comment:
is there a reason why these aren't [instance](https://pynative.com/python-instance-methods/) methods? 
and `log` not an instance variable (potentially injected via the constructor)

globals is usually considered an anti pattern in most programming languages